.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Please briefly explain the changes you made here.
+
+## Jira Ticket
+
+Please add link to the Jira ticket.
+
+## Coding Standards
+
+https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/2108522539/Java+Coding+Standards
```

Comment:
This doesn't feel helpful. As a team we should be following the coding standards, but this is just bloating PR descriptions. We'll end up just automatically ignoring it, like banner-blindness. Maybe this link should be given in the Development section of the README.md?

Reply:
Added to the README